devicetypes/smartthings/springs-window-fashions-shade.src/springs-window-fashions-shade.groovy

```diff
@@ -234,7 +241,14 @@ def close() {
 }
 
 def setLevel(value, duration = null) {
-    log.debug "setLevel(${value.inspect()})"
+    log.debug "setLevel($value)"
```

Comment:
indentation seems off here too

Reply:
Heh Yeah, I wouldn't spend too much time reviewing this right now. There will be changes. I had switched to using Atom, so it seems the indenting crap is still messed up.....